### PR TITLE
Web development - Web Design for Everybody: Basics of Web Development & Coding Specialization   &   Front-End Development

### PR DESCRIPTION
**Pull request by BagchiShreya**
Added a new front-end resource (Coursera course link)

**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

File: Web Development/README.md [hunk 1]
```diff
@@ -104,6 +104,10 @@ This category is a curated collection of valuable web development resources cont
     <td><a href="https://www.youtube.com/watch?v=bMknfKXIFA8">React Course - Beginner's Tutorial for React JavaScript Library [2022]</a></td>
     <td>React course for beginners. It simplifies and demystifies react to any beginner who is willing and eager to learn.</td>
   </tr>
+  <tr>
```
Comment (jfmartinz) [inline Web Development/README.md:107]: This is not suitable to the web development under tutorial or courses, instead you can put it in the [Category Section](https://github.com/jfmartinz/ResourceHub/tree/main/UI-UX%20Design) under [Tutorial or Courses](https://github.com/jfmartinz/ResourceHub/tree/main/UI-UX%20Design#tutorials-or-courses). After these change, I will merged your pr, thank you!

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: Web Development/README.md [hunk 2]
```diff
@@ -104,6 +104,10 @@ This category is a curated collection of valuable web development resources cont
...
+    <td><a href="https://www.coursera.org/specializations/web-design?skipBrowseRedirect=true">Web Design for Everybody: Basics of Web Development & Coding Specialization]</a></td>
```
Comment (jfmartinz) [inline Web Development/README.md:108]: This is not suitable to the web development under tutorial or courses, instead you can put it in the [Category Section](https://github.com/jfmartinz/ResourceHub/tree/main/UI-UX%20Design) under [Tutorial or Courses](https://github.com/jfmartinz/ResourceHub/tree/main/UI-UX%20Design#tutorials-or-courses). After these change, I will merged your pr, thank you!

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Review by jfmartinz — APPROVED** [review]
Looks good

**Comment by jfmartinz** [comment]
One more thing @BagchiShreya before I merged this PR, you need to resolve the conflict, you can check this [link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for more information

**Comment by BagchiShreya** [comment]
> One more thing @BagchiShreya before I merged this PR, you need to resolve the conflict, you can check this [link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for more information

I resolved the conflict

**Pull request merged**